packages/hatchify-core/src/dataTypes/number/index.spec.ts

```diff
@@ -245,7 +399,7 @@ describe("number", () => {
         },
         control: {
           type: "Number",
-          allowNull: undefined,
+          allowNull: undefined, // todo: arthur - should this default to boolean?
```

Comment:
@roy-coder just want to double check this behaviour:
```
number({"primary":true})
number({"autoIncrement":true})
integer({"primary":true})
integer({"autoIncrement":true})
```
set `allowNull: undefined`

is there a reason we are not always setting this to some type of `boolean`?

Reply:
got it!